spec/features/guest_registers_for_courses_spec.rb

```diff
@@ -91,25 +90,24 @@
     fill_in "Last name", with: "You"
     fill_in "Email", with: "YouRock@gmail.com"
 
-    check name: "students[course_registrations][][course_id]", match: :first
-    select "Follow", from: "students[course_registrations][][role]", match: :first
-    check name: "students[course_registrations][][course_id]", match: :second
-    select "Lead", from: "students[course_registrations][][role]", match: :second
-    check name: "students[course_registrations][][course_id]", match: :third
-    select "Lead", from: "students[course_registrations][][role]", match: :third
-    check name: "students[course_registrations][][course_id]", match: :fourth
-    select "Follow", from: "students[course_registrations][][role]", match: :fourth
-    fill_in "Credit Card Number", with: "4242424242424242", match: :first
-    fill_in "card_code", with: "123", match: :first
-    select "1 - January", from: "card_month", match: :first
-    select "2016", from: "card_year", match: :first
-    click_button "Register for #{date.second.strftime('%B')} Classes"
+    all('input[type=checkbox]').each do |checkbox|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:click` as an argument to `each` instead of a block.